src/install/python/python-options/python-agent-unsupported-frameworks.mdx

```diff
@@ -0,0 +1,90 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review the manual initialization steps for unsupported frameworks
+---
+
+It sounds like you're using an unsupported framework, such as mod_wsgi or WebFaction. Here's what you need to know:
```

Comment:
We support both of those though, I'm a bit confused by this section. We have an entire page about each of them.

For examples maybe use like Quart for ASGI, and maybe like Bobo for WSGI? (Genuinely struggled to find a WSGI framework that's at all used that we don't have support for.)